modules/log6x-rn-6.0.8.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-8_{context}"]
+= Logging 6.0.8
+
+This release includes
+//link:https://access.redhat.com/errata/RHSA-2025:3905[RHSA-2025:3905].
+
+
+[id="logging-release-notes-6-0-8-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, time-based stream sharding was not enabled in Loki, which resulted in Loki being unable to save historical data. With this update, the {loki-op} enables time-based stream sharding in Loki, which helps Loki save historical data. (link:https://issues.redhat.com/browse/LOG-7019[LOG-7019])
```

Comment:
This does not match the text from LOG-7019

Reply:
sorry, copy-paste error, fixed now.